tests/data/formats/SPDXJsonExampleEmptyArrays.json

```diff
@@ -0,0 +1,256 @@
+{
+  "comment": "This is a sample spreadsheet",
+  "name": "Sample_Document-V2.1",
+  "documentDescribes": [
+    "SPDXRef-Package"
+  ],
+  "packages": [
+    {
+      "SPDXID" : "SPDXRef-2269-coverlet.collector.3.2.0.nupkg",
+      "checksums" : [ {
+        "algorithm" : "SHA1",
+        "checksumValue" : "0cf7564fcbdee13f6313edd8bc261ca0564a4bf7"
+      } ],
+      "copyrightText" : "NOASSERTION",
+      "description" : "Coverlet is a cross platform code coverage library for .NET, with support for line, branch and method coverage.",
+      "downloadLocation" : "https://api.nuget.org/packages/coverlet.collector.3.2.0.nupkg",
+      "filesAnalyzed" : false,
+      "hasFiles" : [ ],
+      "homepage" : "https://github.com/coverlet-coverage/coverlet",
+      "licenseConcluded" : "(MIT)",
+      "licenseDeclared" : "(MIT)",
+      "licenseInfoFromFiles" : [ ],
+      "name" : "coverlet.collector.3.2.0.nupkg",
+      "originator" : "Organization: tonerdo",
+      "packageFileName" : "coverlet.collector.3.2.0.nupkg",
+      "supplier" : "Organization: tonerdo",
+      "versionInfo" : "3.2.0"
+    },
```

Comment:
I think we could minimize this file as we only want to test the empty lists. A minimal document with only this package and the necessary infromation at document-level would be enough here.